### ISSUE TITLE
Developer greyed options for scenario selection isn't clear on permissions

### ISSUE THREAD

**Issue by Lextuga007** [The-Strategy-Unit/nhp_inputs#732]
From QA and getting a test scenario set up for nhp_outputs there was confusion with the greyed out options for the scenario options as they suggested "no permissions", but really meant "no previous scenario was available". 

This came about particularly because there were multiple sites available from the developer view and although nhp_inputs wasn't being tested it would be useful to test users have correct restrictions.

**Comment by Lextuga007** [comment]
This is possibly a colour and text issue as the text "Scenario name must be completed in order to proceed" is in the same grey as the selection boxes. The colour meaning is not clear on its own as the text is grey regardless of existing scenarios that have been run. 

I suggest that the text is black and moved to above the selection radio buttons. The text could be more explicit to say:

> Scenario options are available where previous runs have been created. To run a new scenario enter the name first. 

<img width="503" height="439" alt="Screenshot of the scenario selection part of the app" src="https://github.com/user-attachments/assets/8aa699db-2a7c-41f2-afd2-3a55d119e7b3" />
